components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -28,8 +36,16 @@ class CustomTabsToolbarFeature(
 
     internal fun initialize(session: Session) {
         session.customTabConfig?.let { config ->
+            // Hide tab counter
+            toolbar.removeAllActions()
+            // Put the toolbar only in display mode
+            toolbar.displayMode()
```

Comment:
Isn't that the default? Did you run into issues if not doing that explicitly?

Reply:
No I didn't, but since there wasn `editMode()` as well, I thought it would be better to be explicit?